src/main/java/cinema/model/MovieSession.java

```diff
@@ -17,17 +14,12 @@
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    @ManyToOne(fetch = FetchType.LAZY)
+    @ManyToOne
     private Movie movie;
-    @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "cinema_hall_id")
+    @ManyToOne
     private CinemaHall cinemaHall;
-    @Column(name = "show_time")
     private LocalDateTime showTime;
```

Comment:
bring back @column, @joincolumn annotations

check all your models, add annotations: @Table, @Column, @JoinTable, @JoinColumn - there should not be any camelCase in your db